org/mozilla/jss/netscape/security/extensions/AuthInfoAccessExtension.java

```diff
@@ -81,13 +81,6 @@
 
     private Vector<AccessDescription> mDesc = new Vector<AccessDescription>();
 
-    /**
-     * Create the extension from the passed DER encoded value of the same.
-     *
-     * @param critical true if the extension is to be treated as critical.
-     * @param value Array of DER encoded bytes of the actual value.
-     * @exception IOException on error.
-     */
```

Comment:
The origin one:
```java
    /**                                                                         
     * Create the extension from the passed DER encoded value of the same.         
     *                                                                          
     * @param critical true if the extension is to be treated as critical.         
     * @param value Array of DER encoded bytes of the actual value.             
     * @exception IOException on error.                                         
     */                                                                         
    public AuthInfoAccessExtension(boolean critical) {                          
        this.extensionId = ID;                                                  
        this.critical = critical;                                               
        this.extensionValue = null; // build this when encodeThis() is called   
    }                                                                           
                                                                                
    public AuthInfoAccessExtension(Boolean critical, Object value)              
            throws IOException {                                                
        this.extensionId = ID;                                                  
        this.critical = critical.booleanValue();                                
        this.extensionValue = ((byte[]) value).clone();                         
        decodeThis();                                                           
    }
```
I've removed doc comment because it doesn't conform the actual code at all ( as I can see). Of course, I will follow you.

Reply:
Done.